tools/tuhi_kete.py

```diff
@@ -562,6 +566,20 @@ def __init__(self, completekey='tab', stdin=None, stdout=None):
         # patching get_names to hide some functions we do not want in the help
         self.get_names = self._filtered_get_names
 
+        try:
+            os.mkdir(CONFIG_PATH)
+        except FileExistsError:
+            pass
+
+        self._history_file = os.path.join(CONFIG_PATH, 'history')
```

Comment:
I pushed this but in half-hindsight I wonder if it should be `histfile` just for googleability. Not that this really matters...

Reply:
OK, better fixing that right now. I'll fix it and push to master.

Reply:
FWIW:
   e63785b..c86c318  master -> master